README.md

```diff
@@ -1,23 +1,52 @@
-# Hanami
+[![Gem Version](https://badge.fury.io/rb/hanami.svg)](http://badge.fury.io/rb/hanami)
+[![Build Status](https://secure.travis-ci.org/hanami/hanami.svg?branch=master)](http://travis-ci.org/hanami/hanami?branch=master)
+[![Coverage](https://coveralls.io/repos/hanami/hanami/badge.svg?branch=master)](https://coveralls.io/r/hanami/hanami)
+[![Code Climate](https://codeclimate.com/github/hanami/hanami.svg)](https://codeclimate.com/github/hanami/hanami)
+[![Dependencies](https://gemnasium.com/hanami/hanami.svg)](https://gemnasium.com/hanami/hanami)
+[![Inline docs](http://inch-ci.org/github/hanami/hanami.svg)](http://inch-ci.org/github/hanami/hanami)
+
+# Hanami :cherry_blossom:
 
 The web, with simplicity.
 
 ## Frameworks
 
-Hanami combines small yet powerful frameworks:
+Hanami is a **full-stack** Ruby web framework.
+It's made up of smaller, single-purpose libraries.
 
-* [**Hanami::Utils**](https://github.com/hanami/utils) - Ruby core extensions and class utilities
-* [**Hanami::Router**](https://github.com/hanami/router) - Rack compatible HTTP router for Ruby
+This repository is for the full-stack framework,
+which provides the glue that ties all the parts together.
+
+Here are the smaller single-purpose frameworks:
```

Comment:
thank you! Actually just got rid of this because we had the same wording a couple lines up